packages/components/button/src/button.js

```diff
@@ -26,6 +26,8 @@ export class TSButton extends TSElement {
 			busy: { type: Boolean, reflect: true },
 			/** Icon name, see the list of available icons in ts-icon component. Also note that it will hide the slot content unless the type is text */
 			icon: { type: String, reflect: true },
+			/** Icon source path. Also note that it will hide the slot content unless the type is text */
```

Comment:
Can we explain a bit about what's the difference between `iconSrc` and `icon` properties here, and when to use which?